src/app.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+/* eslint-disable no-console */
+const numbers = '0123456789';
+const option = process.argv.slice(2);
+const args = process.argv.slice(3).filter(el => numbers.includes(el));
+
+switch (option[0]) {
+  case 'sum':
+    args.length
+      ? console.log(args.map(el => +el)
```

Comment:
It will be good to create a helper function for converting your array type to a number and call it here and in multiply case